readme.md

```diff
@@ -16,22 +16,32 @@ $ npm install --save date-time
 const dateTime = require('date-time');
 
 dateTime();
-//=> '2014-01-09 06:46:01 UTC'
+//=> '2016-07-17 21:03:59 UTC'
 
-dateTime(new Date(2050, 1, 2));
-//=> '2050-02-01 23:00:00 UTC'
+dateTime({date: new Date(1994, 7, 2)});
+//=> '1994-08-02 00:00:00 UTC'
+
+dateTime({local: true});
+//=> '2016-07-17 21:03:59 UTC+0530'
+
+dateTime({date: new Date(1994, 7, 2), local: true});
+//=> '1994-08-02 00:00:00 UTC+0530'
 ```
 
 
 ## API
 
-### dateTime([date])
+### dateTime([{date: Date, local: value}])
```

Comment:
The first argument should still be the data as it's required. Second argument can be the object.